go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -581,9 +587,10 @@ func usersGet(w http.ResponseWriter, r *http.Request) {
 	q := u.Query()
 	userData.queryZoneId = q.Get("zoneId")
 	userData.queryApId = q.Get("accessPointId")
+	userData.queryAddress = q.Get("address")
 
 	// Get user list from DB
-	var response ResponseUserList
+	var response InlineResponse2001
```

Comment:
NOTE: Assuming we wish stick with the currently defined specs with inline object definitions, then we should find a way of making the generated code compliant, clear & maintainable; most likely by modifying the official yaml used for code gen by including $ref with named structures instead of using inline objects.

Reply:
Other options for removing inline structures could include codegen options, mustache config, etc...

Reply:
that;s a big issue on its own... need guidance for this... we do not want to change the spec.. but yes i agree with you regarding how easy it is to maintain... on the other end,,, spec should not be changed often so we can keep it up like this.. although i agree it makes it harder to read... maybe created a dummy file with names of structure like this would help:

type ResponseUserList struct {
        InlineResponse2001
}

opinion?

Reply:
After further discussion, here is what was concluded:

- **Inline Responses:**
  - Must be replaced by references to new structures defined in component **responses**
  - Updates must be done to yaml specification
- **Inline Structs:**
  - Must be replaced by references to new structures defined in component **schemas**
  - Inline structure names must stats with "inline" keyword

Reply:
After further testing, references to structures in component **responses** continues to generate the undesired inline responses. To fix this we need to add these structures to the component **schemas** instead. Also, to minimize code changes with original spec, we should only create inline structures for the unnamed root elements.